core/src/main/scala/org/apache/spark/SparkHadoopWriter.scala

```diff
@@ -23,11 +23,11 @@ import java.text.SimpleDateFormat
 import java.util.{Date, Locale}
 
 import org.apache.hadoop.fs.FileSystem
-import org.apache.hadoop.fs.Path
 import org.apache.hadoop.mapred._
 import org.apache.hadoop.mapreduce.TaskType
 
 import org.apache.spark.internal.Logging
```

Comment:
i'd move this file into spark/internal/io as well to be closer to SparkNewHadoopWriter.scala